common/istio/istio-install/components/ambient-mode/ztunnel.yaml

```diff
@@ -0,0 +1,177 @@
+---
+# Source: ztunnel/templates/rbac.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ztunnel
+  namespace: istio-system
+  labels:
+    app.kubernetes.io/name: ztunnel
+    app.kubernetes.io/managed-by: "Helm"
+    app.kubernetes.io/instance: "ztunnel"
+    app.kubernetes.io/part-of: "istio"
+    app.kubernetes.io/version: "1.27.1"
+    helm.sh/chart: ztunnel-1.27.1
+  annotations:
+    {}
+---
+# Source: ztunnel/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ztunnel
+  namespace: istio-system
+  labels:
+    app.kubernetes.io/name: ztunnel
+    app.kubernetes.io/managed-by: "Helm"
+    app.kubernetes.io/instance: "ztunnel"
+    app.kubernetes.io/part-of: "istio"
+    app.kubernetes.io/version: "1.27.1"
+    helm.sh/chart: ztunnel-1.27.1
+  annotations:
+    {}
+spec:
+  updateStrategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: ztunnel
+  template:
+    metadata:
+      labels:
+        sidecar.istio.io/inject: "false"
+        istio.io/dataplane-mode: none
+        app: ztunnel
+        app.kubernetes.io/name: ztunnel
+        app.kubernetes.io/managed-by: "Helm"
+        app.kubernetes.io/instance: "ztunnel"
+        app.kubernetes.io/part-of: "istio"
+        app.kubernetes.io/version: "1.27.1"
+        helm.sh/chart: ztunnel-1.27.1
+      annotations:
+        sidecar.istio.io/inject: "false"
+        prometheus.io/port: "15020"
+        prometheus.io/scrape: "true"
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      serviceAccountName: ztunnel
+      tolerations:
+      - effect: NoSchedule
+        operator: Exists
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - effect: NoExecute
+        operator: Exists
+      containers:
+      - name: istio-proxy
+        image: "docker.io/istio/ztunnel:1.27.1"
```

Comment:
will this be covered by the istio upgrade script https://github.com/kubeflow/manifests/blob/master/scripts/synchronize-istio-manifests.sh ? CC @kunal-511